net._86box._86Box.ROMs.yaml

```diff
@@ -0,0 +1,34 @@
+id: net._86box._86Box.ROMs
+build-extension: true
+runtime: net._86box._86Box
```

Comment:
This is not a correct runtime.

Reply:
It should be. See for example [here](https://github.com/flathub/com.valvesoftware.Steam.CompatibilityTool.Proton-GE/blob/beta/com.valvesoftware.Steam.CompatibilityTool.Proton.yml#L3).

Reply:
We probably have to wait a little longer before re-running the build. The main net._86box._86Box app is already on Flathub, but there can be some cache issue on the buildbot.

Reply:
Oh, now that you've linked Proton's manifest, I guess we should set `runtime-version: stable`, too. IIRC it defaults to `master` otherwise.

Reply:
That was it, thanks!